storio-content-resolver-annotations-processor/build.gradle

```diff
@@ -1,4 +1,5 @@
 apply plugin: 'java'
+apply plugin: 'com.uber.infer.java'
```

Comment:
Thank you, I just forgot to remove it from here. It doesn't work with annotation processors at all.
```
* What went wrong:
Execution failed for task ':storio-content-resolver-annotations-processor:inferPrepare'.
> Configuration with name 'provided' not found.
```